HTML/mockup-mogo/js/task8.js

```diff
@@ -0,0 +1,15 @@
+import {promiseElements} from './elements.js';
+
+export const showPopup = () => {
```

Comment:
Call `showPopup` means pop up will be shown. But this function just initialise popup, so `initPopup` is more relevant